src/main/java/org/beanpod/switchboard/service/DecoderServiceImpl.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
```

Comment:
can i ask what this import does? 😅

Reply:
Yes, it imports slf4j which is a logging library in Java

Reply:
ahh it generates the log instance! i get it now!